src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,24 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
+    private static final String SMALL_LETTER_W = "w";
+
+    public String[] readFromFile(String fileName) throws RuntimeException {
+        try {
+            List<String> sortedWordsWithW = new ArrayList<>();
+            for (String line : Files.readAllLines(Paths.get(fileName))) {
+                for (String word : line.toLowerCase().split(" ")) {
+                    if (word.substring(0, 1).equals(SMALL_LETTER_W)) {
```

Comment:
there is a predefined method in class String for this purpose

Reply:
Do you mean matches instead of equals? Then I don't need constant - some improvement in code. But if something else, pls let me know. Thanks!!!

Reply:
let's use `word.startsWith(String prefix)` method

Reply:
bring back your constant and use it as a prefix

Reply:
sure = on this case I need constant. But matches looks cleaner:)